react-client/src/Components/DomInfoDialogBox/index.js

```diff
@@ -207,111 +235,131 @@ const initializeDomObserver = async () => {
+                    <div className="dom-styles">Size</div>
+                  </div>
+                  <div className="flex-column">
+                    <div className="dom-styles-details">
+                      <ColorDetails color={textcolor} />
+                    </div>
+                    <div className="dom-styles">Text-Color</div>
+                  </div>
+                  <div className="flex-column">
+                    <div className="dom-styles-details">
+                      <ColorDetails color={backgroundColor} />
+                    </div>
+                    <div className="dom-styles">Background-Color</div>
+                  </div>
                 </div>
+                <div className="dom-styles-details">{fontfamily}</div>
+                <div className="dom-styles">Font Family</div>         
+                <ParentDetails 
+                  tag={parent.tag}
+                  id={parent.id}            
+                  classes={parent.classes}
+                  displayDetails={onClickParent}
+                />
+                <div className="dom-dialog">data-* attributes </div>             
+                <div className="dom-dialog-child-details">   
+                  {dataAttributes.slice(0, numAttibToDisplay).map((val) => (
+                    <div className="attributecontainer">              
+                      <div className="attributeitems">                 
+                        {val.key}                
+                      </div>
+                      <div className="attributeitems">
+                        {val.value}
+                      </div>
+                    </div>
+                  ))}                             
+                </div>
+                {dataAttributes.length > 2 &&(            
+                  <div
+                    id="closeDom"
+                    className="see-more"
+                    onClick={handleSeeMoreAttr}
+                  >
+                    {seeMoreAttr  ? `... ${attrleftover} more` : `... see less`}
+                  </div>
+                )}
+                <ChildrenDetails 
+                  children={domInfo.children}
+                  selectedChildIdx={onClickChild}
+                  />         
+              </div>               
+              <div className="flex-column">
+                {displayParentBox && (              
+                  <DomInfoExtension
+                  extendedDom={domElement.parentElement}
+                  extendedAttributes={domUtils.extractDomInfo(domElement.parentElement).attributes}
+                  showParent={false}
+                />
+                )}
+              </div>
+              <div className="flex-column">
+                {displayChildBox && (                
+                  <DomInfoExtension
+                  extendedDom={selectedChildEl}
+                  extendedAttributes={domUtils.extractDomInfo(selectedChildEl).attributes}
```

Comment:
I don't like how this was implemented for some reason. This poses potential bugs and issues to me.